.circleci/config.yml

```diff
@@ -268,14 +278,12 @@ jobs:
     steps:
       - integration-tests:
           env: alpha
-      - notify-qa
```

Comment:
No longer need CircleCI Slack/orb for `integration-tests-alpha` and `integration-tests-staging` jobs.